Extensions/quick_tags.js

```diff
@@ -354,8 +372,9 @@ XKit.extensions.quick_tags = new Object({
 
 		var user_tag_array = XKit.extensions.quick_tags.load_tag_prefs();
 		var m_user_tags = XKit.extensions.quick_tags.render_tags_from_array(user_tag_array, true);
+		var m_add_button = XKit.extensions.quick_tags.render_add_bundle_button(true);
```

Comment:
Nope, since we only want to add it in the 1-click postage bundle list, and not in the cpanel bundle list.